darwin-module.nix

```diff
@@ -1,12 +1,10 @@
-{ pkgs, databases, ... }:
-
-let
-  nix-index-with-db = pkgs.callPackage ./nix-index-wrapper.nix {
```

Comment:
not using `callPackage` in each module is simpler but it's slightly more likely to cause breakage
but i've seen using the packages through `self` much more

Reply:
I don't think we can use self, because we don't know what specialArgs a user sets up and what nix-index-database is called in the input list.